### DISCUSSION TITLE
Assumption reference-able theorem-like environment

### DISCUSSION THREAD

**Discussion by i-thot-u-wanted2dance** [quarto-dev/quarto-cli#5644] [Feature Requests]
It would be great to add "Assumption" to the list of [supported reference-able theorem-like environments](https://quarto.org/docs/authoring/cross-references.html#theorems-and-proofs).

Some candidates for a suitable label prefix:

- `#asm-`
- `#asp-`
- `#asu-`

**Comment by mcanouil** [comment]
I am closing this since it is already been tracked on #3979 (initially from #3970)

**Reply by cscheid** [reply]
Feel free to refer directly to the [epic issue](https://github.com/quarto-dev/quarto-cli/issues/4944) as well.
